.github/workflows/tests.yml

```diff
@@ -127,4 +127,9 @@ jobs:
       - run: cargo build
       - run: python -m venv testvenv
       - run: ./testvenv/bin/python -m pip install -r tests/requirements.txt
+
+      - name: Run database migrations
+        run: |
+          ./testvenv/bin/python -m alembic.config upgrade head
```

Comment:
At some point, we will need to cleanup the database between gabbits. Shouldn't we move this to tests?

Reply:
That's what I was thinking too. I'm glad we are on the same page about that :) Will do!

Reply:
I've sent #115 !